databases/week2/part3_queries.sql

```diff
@@ -0,0 +1,48 @@
+
+--Part 3: More queries
+--Get all the tasks assigned to users whose email ends in @spotify.com
+use hyf_lesson2;
+
+SELECT task.title, user.email
+from task
+join user_task
+ON task.id = user_task.task_id
+JOIN user
+on user.id = user_task.user_id
+WHERE user.email LIKE "%@spotify.com";
+
+--Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT task.title, status.name
+from task
+join user_task
+on task.id = user_task.task_id
+join status
+on task.status_id = status.id
+WHERE user_task.user_id = 11 AND status.id = 3;
```

Comment:
you need to use 'Donald Duck' with status 'Not started', however you used the ID, please correct it :)